recipes/libpng/all/conanfile.py

```diff
@@ -9,17 +10,27 @@ class LibpngConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://www.libpng.org"
     license = "libpng-2.0"
-    exports_sources = ["CMakeLists.txt", "patches/*"]
-    generators = ["cmake", "cmake_find_package"]
-    settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False], "api_prefix": "ANY"}
-    default_options = {'shared': False, 'fPIC': True, "api_prefix": None}
     topics = ("conan", "png", "libpng")
 
-    _source_subfolder = "source_subfolder"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "api_prefix": "ANY",
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "api_prefix": None,
```

Comment:
Would a default being `""` also be good?

Reply:
Sure, I did that in h3 recipe